### PR TITLE
Fix a couple of javadoc warnings in HA.

### PR DESCRIPTION
**Pull request by chrisvest**

### CODE REVIEW

File: enterprise/ha/src/main/java/org/neo4j/kernel/ha/UpdatePuller.java [hunk 1]
```diff
@@ -228,7 +228,7 @@ public synchronized void unpause()
      * Gets the update puller going, if it's not already going, and waits for the supplied condition to be
      * fulfilled as part of the update pulling happening.
      *
-     * @param condition {@link Condition} to wait for.
+     * @param condition {@link org.neo4j.kernel.ha.UpdatePuller.Condition} to wait for.
```
Comment (thobe) [inline enterprise/ha/src/main/java/org/neo4j/kernel/ha/UpdatePuller.java]: It should suffice to say `UpdatePuller.Condition`

File: enterprise/ha/src/main/java/org/neo4j/kernel/ha/com/master/SlavePriorities.java [hunk 2]
```diff
@@ -62,7 +62,7 @@ public static SlavePriority givenOrder()
     /**
      * @return {@link SlavePriority} which returns the slaves in a round robin
      * fashion, more precisely the start index in the array increments with
-     * each {@link SlavePriority#prioritize(Iterable<Slave>) prioritization}, ordered
+     * each {@link SlavePriority#prioritize(java.lang.Iterable) prioritization}, ordered
```
Comment (thobe) [inline enterprise/ha/src/main/java/org/neo4j/kernel/ha/com/master/SlavePriorities.java]: Does this really need a fully qualified name?

File: enterprise/ha/src/main/java/org/neo4j/kernel/ha/UpdatePuller.java (hunk 1 above)
Comment (chrisvest) [inline enterprise/ha/src/main/java/org/neo4j/kernel/ha/UpdatePuller.java]: Maybe. I let Intellij auto-complete it.

Comment (chrisvest) [inline enterprise/ha/src/main/java/org/neo4j/kernel/ha/UpdatePuller.java]: Yep, looks like you're right.

**Pull request merged**